src/Screens/Gameplay/Components/Settings/styles.js

```diff
@@ -0,0 +1,28 @@
+import { GeneralButton } from 'GeneralComponents/GeneralButton/styles';
+import styled from 'styled-components';
+import iconAsset from 'Assets/settingsIcon.png';
+
+export const Button = styled(GeneralButton)`
+	width: ${({ theme }) => theme.calcHeightUnits(1)};
```

Comment:
changed val to theme.sizes.cuttonsHeight.L, so that simetric W&H are defined at the same space.